tests/container/jetty/src/test/java/org/hawkular/apm/tests/client/jetty/ClientJettyReaderWriterAsyncTest.java

```diff
@@ -270,7 +270,7 @@ protected void testJettyServlet(String method, String urlstr, String reqdata, bo
 
         if (fault) {
             assertEquals("401", testProducer.getFault());
-            assertEquals("Unauthorized", testProducer.getFaultDescription());
+            assertEquals("Unauthorized", testProducer.getDetails().get("fault.description"));
```

Comment:
It would be good to extract this to some Constant class where we can define all keys.